boards/arm/stm32f3_disco/doc/index.rst

```diff
@@ -65,6 +65,7 @@ STM32F3DISCOVERY Discovery kit provides the following hardware components:
 - USB 2.0 full speed interface
 - Infrared transmitter
 - DMA Controller
+- independent watchdog
```

Comment:
I was rather referring to the "Supported features" section. Please update

Reply:
ok, moved